vortex-serde/src/layouts/read/footer.rs

```diff
@@ -85,7 +90,7 @@ impl Footer {
     }
 
     /// Convert all name based references to index based for sake of augmenting read projection
-    pub(crate) fn resolve_references(&self, projection: &[Field]) -> VortexResult<Vec<Field>> {
+    pub fn resolve_references(&self, projection: &[Field]) -> VortexResult<Vec<Field>> {
```

Comment:
This was pub(crate) since it's a weird function that should only be used within this crate. If we are no longer merging filter and output schemas it should be removed